### PR TITLE
convert tabs to code tabs to preserve whitespace

### PR DESCRIPTION
**Pull request by shilohd**
The current version of the duck book does not correctly format the directory structure tabs in the "structuring your project" section. I've changed them to code tabs to avoid sphinx ignoring new lines and whitespace.

### CODE REVIEW

File: book/project_structure.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ A typical analytical project folder might look like:
 |   -- generate_plots.py
 ```
 
-``` {tab} R
+``` {code-tab} r R
```
Comment (joshlynchONS) [inline book/project_structure.md:101]: should this be r and R, or just R?

Comment (shilohd) [reply book/project_structure.md:101]: "r" tells sphinx this is an R code tab. "R" is the title it displays.

Comment (joshlynchONS) [reply book/project_structure.md:101]: All good then

**Review by joshlynchONS — APPROVED** [review]
Everything looks good to me

**Pull request merged**